src/deepsparse/sentence_transformers/sentence_transformer.py

```diff
@@ -265,6 +269,7 @@ def tokenize(
             # No padding needed
```

Comment:
```suggestion
```
Not true anymore

Reply:
Yeah, it should specify it isn't doing static max_length padding though - just padding to the largest sequence in the batch